### ISSUE TITLE
Add Markdown directly to a journal page

### ISSUE THREAD

**Issue by andreoliwa** [andreoliwa/logseq-doctor#17]
## Problem

As a user, I want to add converted Markdown directly to a journal page, in order to avoid copy/paste and manual editing.

## Possible Solution

- [x] Add a `journal` command.
- [x] Receive Markdown data from stdin.
- [x] ~~Or accept files as arguments (like #12).~~
   - won't do, not needed; I can use `cat file1.txt file2.txt | lsd journal`
- [x] Default is to add the markdown to the current date.
- [x] Add `--date` option to allow choosing a date.
- [x] If possible, accept flexible date formats like `yesterday`, `Friday`, `Fri`, `March 27th`, etc.
- [x] Default is to append the Markdown to the end of the journal date.
- [ ] Add option `--prepend` (or something similar) to prepend to the beginning of the journal.
- [x] Try to convert to outline before inserting? This (kind of) depends on #11 being implemented first.
